modules/exploits/linux/http/bludit_upload_images_exec.rb

```diff
@@ -0,0 +1,266 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::PhpEXE
+  include Msf::Exploit::FileDropper
+  include Msf::Auxiliary::Report
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Bluedit Directory Traversal Image File Upload Vulnerability",
```

Comment:
Should this be Bludit?

```suggestion
      'Name'           => "Bludit Directory Traversal Image File Upload Vulnerability",
```